server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -231,4 +243,24 @@ private String convertToString(ArrayList<String> comments) {
 
     return res.toString();
   }
+
+  /**
+   * Filters out a comment string so that it's readable and doesn't break the frontend.
+   *
+   * @param comment The comment to be filtered.
+   * @return A properly formatted comment
+   */
+  private String filterComment(String comment) {
```

Comment:
could this be a method within CommentLikes.java class instead?

Reply:
I like the idea, but the Comment.java class is supposed to represent a data "type". I would prefer to have no methods inside it.